pkg/blueprint/validate/validate.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validate
+
+import (
+	"fmt"
+
+	kanister "github.com/kanisterio/kanister/pkg"
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	_ "github.com/kanisterio/kanister/pkg/function"
+	"github.com/kanisterio/kanister/pkg/param"
+	"github.com/kanisterio/kanister/pkg/utils"
+)
+
+func Do(bp *crv1alpha1.Blueprint) error {
+	for name, action := range bp.Actions {
+		// GetPhases also checks if the function names referred in the action are correct
+		phases, err := kanister.GetPhases(*bp, name, kanister.DefaultVersion, param.TemplateParams{})
+		if err != nil {
+			utils.PrintStage(fmt.Sprintf("Validation of action %s, error %s", name, err.Error()), utils.Fail)
```

Comment:
Do we need to print the error here since we are returning it anyway? The command execution will fail with the error too
```suggestion
			utils.PrintStage(fmt.Sprintf("validation of action %s", name), utils.Fail)
```
The message might end up being something like `Failed the validation of action backup, error Requested function {KubeTasks} has not been registered check.. ❌ ` which might be confusing 🤔